elastica/timestepper/symplectic_steppers.py

```diff
@@ -120,88 +135,46 @@ def do_step(
 
         return time
 
+    def step_single_instance(
+        self: SymplecticStepperProtocol,
+        System: SystemType,
+        time: np.floating,
+        dt: np.floating,
+    ) -> np.floating:
```

Comment:
Why do we need this?

Reply:
This is for testing. We have test cases that test stepping with single objects. I think it would be good idea to keep them in place.